Microsoft.Toolkit.Uwp/Helpers/ColorHelper.cs

```diff
@@ -225,8 +227,7 @@ public static HsvColor ToHsv(this Color color)
                 h1 = 4 + ((r - g) / chroma);
             }
 
-            double lightness = 0.5 * (max - min);
-            double saturation = chroma == 0 ? 0 : chroma / (1 - Math.Abs((2 * lightness) - 1));
+            double saturation = chroma == 0 ? 0 : chroma / max;
```

Comment:
Maybe you should add chroma value check plus optional increment by 6 above if max ==r ?